docs/swagger.yml

```diff
@@ -759,21 +763,23 @@ paths:
             $ref: "#/definitions/ScanJobConnectionResultsOut"
         400:
           description: "Invalid input"
+        401:
+          description: "Not authorized"
         404:
           description: "Scan job connection results not found"
   /jobs/{scan_job_id}/inspection/:
     get:
       tags:
         - "Scan Job"
       summary: "Get the inspection results of existing scan job"
-      description: "Get the inspection results of a scan."
+      description: "Get the inspection results of a scan"
       operationId: "getScanInspectionResults"
       produces:
       - "application/json"
       parameters:
       - name: "scan_job_id"
         in: "path"
-        description: "ID of scan job to retrieve inspection results for"
+        description: "ID of scan job to use in search"
```

Comment:
Same comment as above. We should make the descriptions consistent --> either `ID of scan job used in search` or `ID of scan job to use in search`.